lib/common/common/Jobs.inl

```diff
@@ -52,7 +52,7 @@ namespace JsUtil
     template<class TJobManager, class TJobHolder>
     void JobProcessor::PrioritizeJobAndWait(TJobManager *const manager, const TJobHolder holder, void* function)
     {
-        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>;
+        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>();
```

Comment:
This is actually compile time check and optimized to no-op at runtime. No need to add variable name to pollute local name space. As source code documented, see http://www.stroustrup.com/bs_faq2.html#constraints